beets/library.py

```diff
@@ -546,6 +546,7 @@ class Item(LibModel):
         "work_disambig": types.STRING,
         "arranger": types.STRING,
         "grouping": types.STRING,
+        "subtitle": types.STRING,
```

Comment:
How does this field get populated? Which metadata sources support it? We may want to add some logic for this.

Or maybe you handle this using an external plugin?

Reply:
For the moment, no plugin will populate the field. However, adding the field will allow it to be managed and read/written from/to files.